config/redirects.yml

```diff
@@ -132,30 +132,30 @@ redirects:
   "/explore-my-options/return-to-teaching": "/returning-to-teaching"
   "/explore-my-options/overseas-graduates": "/non-uk-teachers/train-to-teach-in-england-as-an-international-student"
   "/explore-my-options/overseas-applicants": "/non-uk-teachers/train-to-teach-in-england-as-an-international-student"
-  "/funding-and-salary/overview/how-you-will-be-paid": "/funding-your-training"
-  "/funding-my-teacher-training": "/funding-your-training"
-  "/funding-my-teacher-training/extra-financial-support": "/funding-your-training"
-  "/funding-my-teacher-training/tuition-fee-and-maintenance-loans": "/funding-your-training"
-  "/funding-and-salary/overview/scholarships": "/funding-your-training"
-  "/explore-my-options/teacher-training-routes/university-led-training/university-led-undergraduate-training/troops-bursary": "/funding-your-training"
-  "/funding-and-salary/overview/funding-by-subject/funding-for-training-to-teach-english": "/funding-your-training"
-  "/funding-and-salary/overview/funding-by-subject/funding-for-training-to-teach-history": "/funding-your-training"
+  "/funding-and-salary/overview/how-you-will-be-paid": "/funding-and-support"
```

Comment:
I think some of these redirects will need to be changed based on the SEO report but let's do that separately